### PR TITLE
Implementing logging best practices in the project

### PR DESCRIPTION
**Pull request by MobiTikula**
This PR is for implementing logging in the project using best practices.
Code should contain new feature to enable the verbose logging from the run script.

### CODE REVIEW

**Comment by MobiTikula** [comment]
Release notes:
- Implementing logging into the project
- Using dynamic GitHub logging
- Upgrading the logs with best practices

File: living_documentation_generator/utils/utils.py [hunk 1]
```diff
@@ -62,5 +65,5 @@ def set_action_output(name: str, value: str, default_output_path: str = "default
 
 
 def set_action_failed(message: str):
-    print(f"::error::{message}")
+    logger.error("::error:: %s", message)
```
Comment (miroslavpojer) [inline living_documentation_generator/utils/utils.py:68]: I am not sure here if print is not required. It needs to me tested.
Please make a TODO for Integration testing.

**Review by miroslavpojer — COMMENTED** [review]
- [ ] pulled
- [ ] code review

File: action.yml [hunk 2]
```diff
@@ -53,6 +57,7 @@ runs:
         INPUT_REPOSITORIES: ${{ inputs.repositories }}
         INPUT_PROJECT_STATE_MINING: ${{ inputs.project_state_mining }}
         INPUT_PROJECTS_TITLE_FILTER: ${{ inputs.projects_title_filter }}
+        INPUT_VERBOSE: ${{ inputs.verbose-logging }}
```
[the remark this answers is not in the document]
Comment (MobiTikula) [reply action.yml]: Fixed in commit: [d235579](https://github.com/AbsaOSS/living-doc-generator/pull/26/commits/d23557990c0a1b474d9700e23d5ee54f3e2e4806).

File: living_documentation_generator/utils/utils.py (hunk 1 above)
Comment (MobiTikula) [reply living_documentation_generator/utils/utils.py:69]: Todo added in commit: [d235579](https://github.com/AbsaOSS/living-doc-generator/pull/26/commits/d23557990c0a1b474d9700e23d5ee54f3e2e4806).

**Review by miroslavpojer — APPROVED** [review]

**Pull request merged**